style.scss

```diff
@@ -11,144 +12,264 @@ body {
+            font-size: 1.5rem;
+        }
+
+        @media (min-width: $screen-md) {
+            margin: 0 600px 0 50px;
+            font-size: 2rem;
+        }
+
+        @media (min-width: $screen-lg) {
+            margin: 0 600px 0 50px;
+            font-size: 2rem;
+        }
+    }
+
+    .main-title {
+        color: $primary;
+        text-align: center;
+        text-wrap: balance;
+        margin-bottom: 1rem;
+        font-size: 1rem;
+
+        @media (min-width: $screen-xs) {
+            font-size: 2rem;
+        }
+    }
+
+    .cards-container {
+        display: flex;
+        flex-direction: column;
+
+        @media (min-width: $screen-xs) {
+            flex-direction: row;
+            flex-wrap: wrap;
+        }
+    }
+
+    .card {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: 1rem;
+        gap: 1rem;
+        background-color: $light;
+        color: $dark;
+
+        @media (min-width: $screen-xs) {
+            width: 50%;
+        }
+
+        @media (min-width: $screen-md) {
+            width: 33.3333333%;
+        }
+
+        &:nth-of-type(odd) {
+            background-color: $secondary-light;
+            color: $light;
+
+            @media (min-width: $screen-xs) {
+                background-color: $light;
+                color: $dark;
+            }
+
+            @media (min-width: $screen-md) {
+                background-color: $secondary-light;
+                color: $light;
+            }
+        }
+
+        &:nth-of-type(1),
+        &:nth-of-type(4),
+        &:nth-of-type(5) {
+            @media (min-width: $screen-xs) and (max-width: ($screen-md - 1)) {
+                background-color: $secondary-light;
+                color: $light;
+            }
+        }
+
+        .icon {
+            width: 50px;
+            height: 50px;
+        }
+
+        .title {
+            text-align: center;
+            font-weight: 400;
+            margin: 0;
+            font-size: 0.8rem;
+
+            @media (min-width: $screen-xs) {
+                font-size: 1rem;
+            }
+        }
+    }
 }
 
-.footer {
-    background-color: $primary-lighter;
+footer {
+    background-color: $primary-light;
     display: flex;
-    flex-direction: row;
     justify-content: center;
-}
 
-.footer-container {
-    display: flex;
-    flex-direction: column;
-    text-align: center;
-}
+    .container {
+        display: flex;
+        flex-direction: column;
+        text-align: center;
+    }
 
-.footer-logo {
-    width: 200px;
-}
+    .logo {
+        max-width: 50px;
 
-.tel-footer {
-    font-size: 30px;
-    font-weight: bold;
-    margin: 15px;
-}
+        @media (min-width: $screen-xs) {
+            max-width: 75px;
+        }
+
+        @media (min-width: $screen-md) {
+            max-width: 100px;
+        }
+    }
+
+    .tel {
+        font-weight: 700;
+        gap: 1rem;
+        margin: 0.5rem;
```

Comment:
Please remove `margin` from this selector and apply it to `footer`. Also, move `gap` to `footer` as well. Always test you changes in browser: `gap` rule is not applicable in this case, and is ignored by browser.